contract-tests/test_api.py

```diff
@@ -165,3 +165,106 @@ def test_recipe_history(conf, requests_session):
             created = datetime.strptime(revision["date_created"], "%Y-%m-%dT%H:%M:%S.%fZ")
             assert created < last_date
             last_date = created
+
+
+def test_docs(conf, requests_session):
+    r = requests_session.get(conf.getoption("server") + "/api/docs")
```

Comment:
This URL (and all the others in the tests) should end in `/`. As it stands, every test is going to be redirecting to the with-slash version of the URL.